vng_api_common/schema.py

```diff
@@ -135,3 +174,16 @@ def get(self, request, version="", *args, **kwargs):
             server["url"] = request.build_absolute_uri(server_path)
 
         return Response(data=schema, headers={"X-OAS-Version": schema["openapi"]})
+
+
+class YamlView(View):
```

Comment:
I don't see a need to serve it, the unresolved version of the YAML file is just for people who want to download the spec, which can be done via Github. Let's remove the view.

Reply:
the view is removed